lib/matplotlib/tests/test_rcparams.py

```diff
@@ -307,7 +307,7 @@ def generate_validator_testcases(valid):
                      ('AABBCC', '#AABBCC'),  # RGB hex code
                      ('AABBCC00', '#AABBCC00'),  # RGBA hex code
                      ('tab:blue', 'tab:blue'),  # named color
-                     ('C0', 'C0'),  # color from cycle
+                     ('C123', 'C123'),  # color from cycle
```

Comment:
Would just (or additionally) test "C12", which is a pontential ambiguous definition, which is more likely to break when changing the parser code.

Reply:
changed